findagig-batch/src/main/java/findagig/source/entity/Event.java

```diff
@@ -93,4 +93,19 @@ public void setArtists(final List<Artist> artists) {
     public enum Type {
         CONCERT, FESTIVAL
     }
+
+    @Override
+    public String toString() {
+        return "Event{" +
```

Comment:
Bad practice here ... (String concatenation)
A better alternative would be use `StringBuilder`.
When asking for Intellij to create toString, it offers some builtin ways to do it.

Reply:
ok. All Entities have been reviewed and modified. String concatenation has been replaced by StringBuilder.